apps/base-docs/tutorials/docs/6_subquery.md

```diff
@@ -0,0 +1,364 @@
+---
+title: 'Indexing Data with SubQuery'
+slug: /subquery
+description: A tutorial explaining SubQuery and providing swift guidance on configuring a project
+author: subquery
+keywords:
+  [
+    indexers,
+    RPC,
+    API,
+    graphql,
+    frontend,
+    smart contract development,
+    EVM,
+    Base,
+    Base network,
+    Base node providers,
+    Base providers,
+    blockchain development,
+    dApps,
+    smart contracts,
+    blockchain API
+  ]
+tags: ['smart contracts', 'api']
+difficulty: beginner
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+## Objectives
+
+- Present the concept of indexers
+- Begin effectively with all types of SubQuery indexer configuration files
+- Successfully build your project
+- Efficiently retrieve data from indexed sources
+
+The goal is to index the all the claims from the [Bridge to Base NFT contract](https://basescan.org/token/0xEa2a41c02fA86A4901826615F9796e603C6a4491) on [Base Mainnet](https://docs.base.org/using-base/).
+
+---
+
+:::info
+
+The final code of this project can be found [here](https://github.com/subquery/ethereum-subql-starter/tree/main/Base/base-nft).
+
+:::
+
+---
+
+The contents presented in this tutorial were discussed previously during the workshop. You can follow along to receive detailed audio commentary on the setup's intricacies:
```

Comment:
Most learners are going to read the article OR watch the video, but not both.  Please make sure they both have the full content!  I can't watch this now but intend to later!